src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--5 .stars__star,
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The ordering of these selectors is not logical. It is better to order them in ascending or descending order for better readability. For example, start from `.stars--1` and end at `.stars--5` or vice versa.